Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1811,6 +1778,80 @@ void CGContextEOFillPath(CGContextRef context) {
 }
 #pragma endregion
 
+#pragma region Drawing Operations - CGImage
+
+/**
+ @Status Interoperable
+*/
+void CGContextDrawImage(CGContextRef context, CGRect rect, CGImageRef image) {
+    NOISY_RETURN_IF_NULL(context);
+    RETURN_IF(!image);
+
+    // Obtain the pixel format for the image
+    WICPixelFormatGUID imagePixelFormat = _CGImageGetPixelFormat(image);
+
+    CFRetain(image);
+    woc::unique_cf<CGImageRef> refImage;
+    refImage.reset(image);
+
+    if (!_CGIsValidRenderTargetPixelFormat(imagePixelFormat)) {
+        // convert it to a valid pixelformat
+        refImage.reset(_CGImageCreateCopyWithPixelFormat(image, GUID_WICPixelFormat32bppPBGRA));
```

Comment:
we set the displaytexture to generate PBGRA in the compositor., Needs an update 

---
In reply to: [88731112](https://github.com/Microsoft/WinObjC/pull/1414#discussion_r88731112) [](ancestors = 88731112)